quinn-proto/src/connection.rs

```diff
@@ -267,30 +267,48 @@ impl ::std::iter::FromIterator<Retransmits> for Retransmits {
     }
 }
 
+pub enum ConnectionConfig<'a> {
```

Comment:
I would prefer to add this code after `Connection` (per the idea that the highest-level API should live on top, and thus be more accessible). What are your thoughts on that?

Reply:
Makes sense to me.